extensions/console/internal.d.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+/// <reference no-default-lib="true" />
+/// <reference lib="esnext" />
+
+declare namespace globalThis {
```

Comment:
What's this? we've never strongly typed internals before.

Reply:
There are other internal.d.ts files. I was gradually typing here.

Reply:
Ah nope nevermind, we do and I'm wrong.